### PR TITLE
Fixed #236: Incorrect data from ws.trades

### LINKED ISSUE #236: Incorrect data from ws.trades
According to [official documentation](https://github.com/binance-exchange/binance-official-api-docs/blob/master/web-socket-streams.md#trade-streams) websocket trade is:

```js
{
  "e": "trade",     // Event type
  "E": 123456789,   // Event time
  "s": "BNBBTC",    // Symbol
  "t": 12345,       // Trade ID
  "p": "0.001",     // Price
  "q": "100",       // Quantity
  "b": 88,          // Buyer order ID
  "a": 50,          // Seller order ID
  "T": 123456785,   // Trade time
  "m": true,        // Is the buyer the market maker?
  "M": true         // Ignore
}
```

But it is [parsed](https://github.com/Ashlar/binance-api-node/blob/master/src/websocket.js#L172) as
```js
w.onmessage = msg => {
      const {
        e: eventType,
        E: eventTime,
        s: symbol,
        p: price,
        q: quantity,
        m: maker,
        M: isBuyerMaker,
        a: tradeId,
      } = JSON.parse(msg.data)
```

Field `tradeId` should point to `t` and no fields `a` (seller orderID) and `b` (buyer orderID)

### PR DESCRIPTION
**Pull request by alexeymarunin**

### CODE REVIEW

**Comment by alexeymarunin** [comment]
@balthazar 
I moved some code to separate function to avoid overhead if/else

**Comment by alexeymarunin** [comment]
Any troubles?

File: test/utils.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const checkFields = (t, object, fields) => {
 }
 
 const generatePort = (() => {
-  let portNum = 9000;
+  let portNum = parseInt(9000 + (65535 - 9000) * Math.random(), 10);
```
Comment (balthazar) [inline test/utils.js]: Do you mind removing this? It seems out of scope of this PR

Comment (alexeymarunin) [reply test/utils.js]: My bad, on my PC ports 9000-9001 are busy by [Portainer](https://www.portainer.io/), so I couldn't run tests (before commits)
I rolled it back

**Comment by balthazar** [comment]
Thank you!

**Pull request merged**